mmv1/third_party/terraform/services/dataproc/resource_dataproc_cluster.go

```diff
@@ -1304,6 +1310,29 @@ func instanceConfigSchema(parent string) *schema.Schema {
 	}
 }
 
+func masterInstanceConfigSchema() *schema.Schema {
+	master_keys := instanceConfigSchemaKeys("master_config")
+	return instanceConfigSchema("master_config", master_keys)
+}
+
+// min_num_instances will be picked up only by primary worker instance groups
+// append elem min_num_instances in primary worker instance groups schemas
+func primaryWorkerInstanceConfigSchema() *schema.Schema {
+	worker_keys := instanceConfigSchemaKeys("worker_config")
+	worker_schema := instanceConfigSchema("worker_config", worker_keys)
+	elements_elem := worker_schema.Elem.(*schema.Resource)
+	elements_elem.Schema["min_num_instances"] = &schema.Schema{
```

Comment:
Note: This is a handwritten resource so we'll need to update the documentation by hand (https://googlecloudplatform.github.io/magic-modules/develop/resource/#add-documentation)